### PR TITLE
Upgrade Wordpress, WooCommerce and plugin

### PR DESCRIPTION
**Pull request by dennisreimann**
Let's also try to make the `RUN ln -s /usr/include/x86_64-linux-gnu/gmp.h /usr/local/include/` line in the Dockerfile more flexible, so that we can support all the different architectures.

### CODE REVIEW

**Review by ndeet — APPROVED** [review]

**Comment by dennisreimann** [comment]
According to a conversion that @ndeet and I had, we might be able to remove GMP, because the new plugin version does not make use of it. This way we could also support the different architectures that the [Wordpress 6.0.2 image](https://hub.docker.com/layers/library/wordpress/6.0.2/images/sha256-7085fbc7c68e5dec71c3bfc4f2d6379846116cdb63961eac3a65f208083e4634?context=explore) offers.

**Comment by ndeet** [comment]
Can confirm it works, tested invoice creation and webhooks on a testnet site with my [image based on Dennis's branch](https://hub.docker.com/layers/ndeet/docker-woocommerce/1.1.1/images/sha256-36f42198dfc2564bc60995f129188b12375f96859721e0da14c27b4b77bcb2ad?context=explore).

**Comment by dennisreimann** [comment]
Inviting @NicolasDorier to also do a quick check and merge this, I'll then take care of updating the [WooCommerce docker fragment](https://github.com/btcpayserver/btcpayserver-docker/blob/master/docker-compose-generator/docker-fragments/opt-add-woocommerce.yml).

**Comment by NicolasDorier** [comment]
@ndeet @dennisreimann can you use this occasion to deprecate this repo and move the dockerifle to https://github.com/btcpayserver/dockerfile-deps ? The CI there already has the logic for multi arch builds.

**Comment by dennisreimann** [comment]
Ok, let's merge and I will transfer it then, so that we can do the release on the other repo.

**Comment by ndeet** [comment]
@dennisreimann Because of a discussion in the chat, I think it would be good to also bump the PHP version. For some reason their default is still PHP 7.4 which will be unsupported as of November 28th this year and not get any security fixes anymore (there will be backports maybe) but many plugins and even wordpress encourages users to go to a newer version.

Latest version is PHP 8.1 but we might want to go with 8.0 (supported until 26th Nov 2023) for the next couple of months before switching to 8.1. 

For going with 8.0 you would need to change first line to `FROM wordpress:6.0.2-php8.0`.

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
-FROM wordpress:6.0.0
+FROM wordpress:6.0.2
```
Comment (dennisreimann) [inline Dockerfile]:
```suggestion
FROM wordpress:6.0.2-php8.0
```

**Pull request merged**